### PR TITLE
Updated local and Netlify deployment to use dev instance as API

### PR DESCRIPTION
**Pull request by karelklima**

### CODE REVIEW

**Review by psiotwo — APPROVED** [review]

File: utils/components.yml [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
 al-ontographer:
   name: ontoGrapher
-  url: https://slovník.gov.cz/modelujeme/v-nástroji/ontographer
+  url: https://xn--slovnk-test-scb.mvcr.gov.cz/modelujeme/v-nástroji/ontographer
   meta:
     workspace-path: '/?workspace=%WORKSPACE_IRI%'
 al-termit-server:
   name: TermIT backend
-  url: http://localhost/modelujeme/sluzby/termit-server
+  url: hhttps://xn--slovnk-test-scb.mvcr.gov.cz/modelujeme/sluzby/termit-server
```
Comment (psiotwo) [inline utils/components.yml:20]: there is a typo hhttps

File: utils/components.yml [hunk 2]
```diff
@@ -1,32 +1,32 @@
 al-sgov-server:
   name: SGoV Service
-  url: https://kbss.felk.cvut.cz/sgov-server
+  url: https://xn--slovnk-test-scb.mvcr.gov.cz/modelujeme/sluzby/sgov-server
   meta: {}
 al-db-server:
   name: A graph database
   url: https://example.com/sparql
```
Comment (psiotwo) [inline utils/components.yml:7]: Not sure, why you replaced termit-server URL, but not this one. Would propose to have everything pointed to slovník-test

**Review by psiotwo — CHANGES_REQUESTED** [review]
see comments

File: utils/components.yml (hunk 1 above)
Comment (karelklima) [reply utils/components.yml]: Fixed

File: utils/components.yml (hunk 2 above)
Comment (karelklima) [reply utils/components.yml]: Fixed. It does not really matter as MC does not access neither of those services. The configuration is applied only for local development and on Netlify, or to be precise for generating local env vars. On the main deployment instances it is always replaced by the docker compose configuration.

**Pull request merged**